tar/bsdtar.c

```diff
@@ -999,18 +999,24 @@ main(int argc, char **argv)
 	if (bsdtar->mode == OPTION_VERIFY_CONFIG)
 		exit(0);
 
+	/*
+	 * Special case: if we're doing a dryrun and the keyfile came
+	 * from a config file, ignore a non-readable keyfile.
+	 */
+	if (bsdtar->keyfile && bsdtar->keyfile_from_config &&
+	    bsdtar->option_dryrun && access(bsdtar->keyfile, F_OK)) {
```

Comment:
Is `F_OK` what we want here?  I'm not sure what the most sensible response is if a key file exists but is not readable.

Either way, it's not consistent with the comment...